_posts/2024-01-19-spring-2024-meeting-announcement.markdown

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "PyHC Spring 2024 Meeting Dates Announced"
+author: jibarnum
+---
+
+PyHC serves as a community knowledge base for performing heliophysics research in Python, aiming to provide a variety of tutorials and resources, a list of useful packages, general discussion, and advice. Twice yearly, PyHC meets in person. These meetings are open to anyone, from individuals already utilizing Python heavily in developing open-source heliophysics packages to those who are just getting into using Python and open source development.
+
+**PyHC's first meeting of 2024 will be held remotely on Zoom. The meeting will run Monday, March 11th, 2024 – Thursday, March 14th, 2024, from 9 AM – 11 AM MT each day.** _The Zoom sessions will be recorded._ Expect excellent discussion opportunities, core project updates, and other related talks and demos! See [the full agenda](https://docs.google.com/spreadsheets/d/1FpjjlXckBGXBVSEt9lEClNf_ORbnGOrEzKd5av_cFXY/edit?usp=sharing) for dates/times of the various sessions. The agenda is currently being finalized, and thus is subject to change.
+
+The meeting's web page can be found [here]({% link _pages/meetings/community_meetings/spring2024.md %}). **Registration—which is free but required—can be found [here](https://forms.gle/WAB9fpiFygdBZo7n6).** The meeting's web page also has a link to registration, as well as the Zoom telecon information, links to important documents, etc. If you'd like to join our mailing list to learn more information about the meeting and other upcoming events, please see the [PyHC Contact page](http://heliopython.org/contact/) for instructions. You'll also find links there for joining our Element and Helionauts chat groups (where we discuss various PyHC issues/topics, as well as discussing other Python-related questions), as well as a link to connect with PyHC on Twitter!
```

Comment:
The link can be made `_pages/meetings/spring2024.md` if you just give the page that permalink when you create it. Please update to match the rest of our URLs.

Reply:
sounds good. Removed and will put correct link in once spr '23 page is live.